lib/ansible/modules/cloud/amazon/elb_classic_lb_info.py

```diff
@@ -165,7 +165,12 @@
 @AWSRetry.backoff(tries=5, delay=5, backoff=2.0)
 def list_elbs(connection, names):
     paginator = connection.get_paginator('describe_load_balancers')
-    load_balancers = paginator.paginate(LoadBalancerNames=names).build_full_result().get('LoadBalancerDescriptions', [])
+    try:
+        load_balancers = paginator.paginate(LoadBalancerNames=names).build_full_result().get('LoadBalancerDescriptions', [])
+    except botocore.exceptions.ClientError as e:
```

Comment:
This will drop most ClientErrors on the floor and block the retries

It would be better to use the helper from ansible.module_utils.aws.core

```
except is_boto3_error_code('LoadBalancerNotFound'):
    return []
```

Reply:
Alternatively, you could reraise after your if.